playbooks/roles/tower/tasks/bootstrap.yml

```diff
@@ -2,6 +2,18 @@
 # Authenticate with tower
 - include_tasks: authenticate_tower.yml
 
+- name: Upload license file "https://{{ tower_host }}"
+  uri:
+    url: "https://{{ tower_host }}/api/v2/config/"
+    return_content: yes
+    user: "{{ tower_username }}"
+    password: "{{ tower_password }}"
+    method: POST
+    force_basic_auth: yes
+    body_format: json
+    body: '{{ tower_license }}'
+  when: tower_license and tower_license != ''
```

Comment:
The `tower_license` var should always be defined given that it's in `defaults/main.yml` of the tower role. I'm wondering if `when: tower_license != ''` would be enough?

Reply:
Yep. Makes sense. I will update that and squash the commits here.